### ISSUE TITLE
moving to Typescript

### ISSUE THREAD

**Issue by pedrobergamini** [viromedia/viro#850]
Are there any plans on moving the library to Typescript? I believe it would be a great improvement

**Comment by jpudysz** [comment]
I don't think so as library is not maintained by viro team anymore

**Comment by yashukla47** [comment]
where did you get this information from? is it not active anymore?

**Comment by ranjanpoudel1234** [comment]
The original developers cannot keep up with so many devices and updates, hence they are asking community to chime in. Thats why they made this opensource. They are still around to answer any critical questions we have especially regarding their architecture, but, if we love this project, it is on us to revive it. Come join the group on this slack https://join.slack.com/t/virodevelopers/shared_invite/zt-46e9uxg7-eiiZLdoLtMhHvTKoX2~L4g, join the roundtable meetings that @RoyceLeonD has put together and lets unite to rock this together from there.

**Comment by yashukla47** [comment]
Definitely, Our team has decided to go with Viro And would definitely try to contribute as much as we can and will try to revive it along the way.

**Comment by ranjanpoudel1234** [comment]
Cheers @yashukla47 , lets do it!

**Comment by arielconti10** [comment]
I'm gonna fork it with TS support
